Improved testing

mhendrey opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

mhendrey committed:
Added pytest & bumped version to 1.2.0

mhendrey committed:
Figuring out how to run pytest in the CI/CD pipeline

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

It appears you are making a pull request from a branch in your feedstock and not a fork. This procedure will generate a separate build for each push to the branch and is thus not allowed. See our [documentation](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests) for more details.

Please close this pull request and remake it from a fork of this feedstock.

Have a great day!

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

mhendrey committed:
Still trying to determine the root directory that pytest is running from

mhendrey committed:
Debugging how to run pytest when building package in conda-forge

mhendrey committed:
Still getting "Tests failed", but can't see what the actual failed tests are

mhendrey committed:
Pipeline sure didn't like the "/" at the end of tests

Tried to use "pytest tests/" as a test command

mhendrey committed:
Had the "pytest tests/" in the wrong place.

Put it in requirements and not commands

mhendrey committed:
Trying to just run "pytest" as the command again

Hoping to find the error

mhendrey committed:
Fingers crossed

I'm following the example in conda-forge.org/docs/maintainer/adding_pkgs.html#test
